### PR TITLE
bump keycloak-operator and integreatly-operator

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation
This is required for creating a [Github](https://github.com/integr8ly/integreatly-operator/pull/61/files#diff-329c149b9583e9a4fcdb670a2ba866f0R316-R317) identity provider to be used for Launcher & Che in OpenShift 4.

## What
This adds the properties `linkOnly` to the [Keycloak Identity Provider](https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_identityproviderrepresentation) type.

## Why
The Github identity provider created for Launcher and Che in OpenShift 4 requires the `Account Linking Only` to be set to true.

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: RHSSO (1.8.1) and Integreatly Operator (1.5.9) should be available in your app registry
2. Create an `OperatorSource` with your app registry
e.g.
```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators austincunningham
  endpoint: 'https://quay.io/cnr'
  publisher: austincunningham Publisher
  registryNamespace: austincunningham
  type: appregistry
```
 - **Verify**: RHSSO should become available in the Operator Hub
 - **Verify**: RHSSO CR should be available in the developer catalog
3. Install RHSSO to a specific namespace
 - **Verify**: An Install plan should be created
 - **Verify**: RHSSO operator should be deployed successfully

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] Changes have been successfully verified by another team member

## Additional Notes
Related PR: https://github.com/integr8ly/integreatly-operator/pull/61 and https://github.com/integr8ly/keycloak-operator/pull/90

### CODE REVIEW

**Comment by matskiv** [comment]
@austincunningham you need to copy the folder that contains csv and crd to keep previous version + in new version you need to update `replaces` attribute in the csv:
https://github.com/integr8ly/manifests/blob/master/integreatly/integreatly-1.5.7/integreatly-operator.1.5.7.clusterserviceversion.yaml#L21

This is due to our OSD instance, this way it will get "upgraded" automatically. Check other recent PRs if in doubt :)

File: integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.5.8
+  maturity: alpha
+  replaces: integreatlyoperator.1.5.7
```
Comment (austincunningham) [inline integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml]: @matskiv a little confused , taught I had that done here , is there somewhere else I need to change it ??

Comment (matskiv) [reply integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml]: @austincunningham You are right. Sorry for the the confusion. I looked just on keycloak-operator version bump, and it needs to be handled the same way as you have done with integreatly-operator.

Comment (austincunningham) [reply integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml]: So keycloak-operator needs previous versions directory as well like so ?
image

File: integreatly-rhsso/rhsso-1.8.1/keycloakoperator.1.8.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -19,7 +19,6 @@ spec:
   description: Red Hat Single Sign-On provides Web single sign-on and identity federation based on SAML 2.0, OpenID Connect and OAuth 2.0 specifications.
   keywords: ['RHSSO', 'keycloak', 'aerogear']
   version: 1.8.1
-  replaces: keycloakoperator.1.8.0
```
Comment (JameelB) [inline integreatly-rhsso/rhsso-1.8.1/keycloakoperator.1.8.1.clusterserviceversion.yaml]: This doesn't work even if you point it to `1.7.4`?

File: integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml (hunk 1 above)
Comment (matskiv) [reply integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml]: yes :)

**Comment by matskiv** [comment]
btw: don't forget to update the version in `manifests/integreatly-rhsso/rhsso.package.yaml`

**Comment by matskiv** [comment]
integreatly-operator 1.5.9 doens't exist yet, once this is pushed to integreatly app registry on quay, integreatly-operator installation via OLM will be failing on image pull error

**Pull request merged**